### ISSUE TITLE
Why doesn't training a model this way work well?

### ISSUE THREAD

**Issue by dengg1013** [VinAIResearch/MISCA#3]
I first train the base model using bert backbone,the following command is
python main.py --token_level word-level
--model_type bert
--model_dir dir_base
--task my dataset
--data_dir data
--attention_mode label
--do_train
--do_eval
--num_intent_detection
--use_crf,
and then loads dir_base model,the following command is
python main.py --token_level word-level
--model_type bert
--model_dir misca
--task my dataset
--data_dir data
--attention_mode label
--do_train
--do_eval
--num_intent_detection
--use_crf \
--base_model dir_base
--intent_slot_attn_type coattention,
however, the result still low.
![image](https://github.com/VinAIResearch/MISCA/assets/121504378/a37db4d0-b954-49fd-8f6a-051f346a89ae)

**Comment by dengg1013** [comment]
maybe the epochs is too few to train the model？

**Comment by dengg1013** [comment]
RUN configuration：
--token_level
word-level
--model_type
lstm
--model_dir
dir_base
--task
all
--data_dir
data
--attention_mode
label
--do_train
--do_eval
--num_intent_detection
--use_crf
**I use lstm encoder ,the result is great as following:
![image](https://github.com/VinAIResearch/MISCA/assets/121504378/083c48e0-1f24-4ea7-a2d5-ed3cb9eefd1e)

But got low score using bert model（train base model first,then train misca）,why?**

**Comment by thinhphp** [comment]
Hi, 
Thanks for your interest! We have updated the default hyper-parameter settings in the README. When training with BERT model, we need to scale down the learning rate (around 1e-5). So, we have updated this setting.
Thanks,

**Comment by tmrnvcome** [comment]
hi @dengg1013  , i tried using the code u shared, but am still experiencing this same error. May I know if you can assist pleasE? 
[
![Screenshot from 2024-02-08 19-06-51](https://github.com/VinAIResearch/MISCA/assets/134356965/cd8ffb1a-9b2d-4877-80f4-d811bd1de077)
](url)
